clouddriver-kubernetes/src/test/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/deployer/KubernetesDeploymentHandlerSpec.groovy

```diff
@@ -141,4 +145,43 @@ spec:
     then:
     result.findAll { a -> a.getReference() == CONFIG_MAP_VOLUME && a.getType() == ArtifactTypes.KUBERNETES_CONFIG_MAP.toString()}.size() == 1
   }
+
+
+  void "check that secret volume is replaced by the artifact replacer"() {
+    when:
+    def target = "$SECRET_ENV-version-1.2.3"
+    def artifact = Artifact.builder()
+        .type(ArtifactTypes.KUBERNETES_SECRET.toString())
+        .name(SECRET_ENV)
+        .reference(target)
+        .build()
+
+    def result = handler.replaceArtifacts(stringToManifest(BASIC_DEPLOYMENT), [artifact])
+
+    then:
+    result.manifest.spec.template.spec.containers[0].envFrom[0].secretRef.name == target
+  }
+
+  void "check that secret volume replaced by the artifact replacer"() {
```

Comment:
Is there an 'only' or 'not' missing from the test case name?

Reply:
yes